hail/python/hail/expr/types.py

```diff
@@ -653,9 +653,16 @@ def _convert_from_json(self, x):
 
     def _convert_to_json(self, x):
         data = x.reshape(x.size).tolist()
+
+        strides = []
+        running_shape_product = x.itemsize
+        for shape_element in reversed(x.shape):
+            strides.insert(0, running_shape_product)
+            running_shape_product = running_shape_product * (shape_element if shape_element > 0 else 1)
```

Comment:
use `*=`

Reply:
Agh, clearly I've been spending too much time in the emitter